packages/blaze/builtins.js

```diff
@@ -226,6 +226,11 @@ Blaze.Each = function (argFunc, contentFunc, elseFunc) {
         });
       },
       changedAt: function (id, newItem, oldItem, index) {
```

Comment:
The only difference with what I did is that it was server side update with method. If you can replicate it, it means that the problem is in the publish / diff-ing.